pkg/operator/staticpod/controllers.go

```diff
@@ -124,6 +129,15 @@ func (b *staticPodOperatorControllerBuilder) WithCerts(certDir string, certConfi
 
 func (b *staticPodOperatorControllerBuilder) WithInstaller(command []string) Builder {
 	b.installCommand = command
+	b.installerPodMutationFunc = func(pod *corev1.Pod, nodeName string, operatorSpec *operatorv1.StaticPodOperatorSpec, revision int32) error {
+		return nil
+	}
+	return b
+}
+
+func (b *staticPodOperatorControllerBuilder) WithMutatingInstallerPod(command []string, installerPodMutationFunc installer.InstallerPodMutationFunc) Builder {
```

Comment:
what does this do? It needs docs.

Reply:
MutatingInstallerPod sounds like the pod is mutating something. In reality we pass a func which mutates, right? `WithMutatedInstaller` or `WithCustomInstaller`?

Reply:
@sttts yes, WithCustomInstaller with do to explain makes more sense, will update